src/test/java/org/akhq/models/decorators/AvroWireFormattedRecordTest.java

```diff
@@ -90,7 +99,10 @@ public void convertValueToWireFormatWireFormat() {
 
         ConsumerRecord<byte[], byte[]> consumerRecord = new ConsumerRecord<>("topic", 1, 0, new byte[0], avroPayload);
         consumerRecord.headers().add(new RecordHeader("contentType", "application/vnd.mySubject.v1+avro".getBytes()));
-        byte[] convertedValue = avroWireFormatConverter.convertValueToWireFormat(consumerRecord, schemaRegistryClient, SchemaRegistryType.CONFLUENT);
+        Record akhqRecord = new Record(consumerRecord, null, null);
+        AvroContentTypeMetaData metaData = AvroContentTypeMetaData.of("mySubject", 1);
+        AvroWireFormattedRecord underTest = new AvroWireFormattedRecord(akhqRecord, schemaRegistryClient, metaData, SchemaRegistryType.CONFLUENT.getMagicByte());
+        byte[] convertedValue = underTest.getBytesValue();
```

Comment:
wenn wir ganz konsequent wären müssten wir auch underTest.getBytesKey() usw. testen, oder? und, ob sich der decorator dekorieren lässt bzw. auch noch funktioniert wenn er einen anderen decorator dekoriert? *hmm*

Siehe https://github.com/Grishnackh/akhq/pull/13/files#diff-c5fa2eff85bb7e38e7f9f4c666238c38b9c96c1ae20a1cbc8c5d0e1f253c04e0R23

Reply:
Absolut, aber sehe ich dann eher im `DecoratorTest`, oder?

Reply:
`DecoratorTest` hatte ich mal angelegt für sowas, ja :) Muss aber gestehen, dass ich mir da nicht so viele Gedanken darüber gemacht hab, wie das sich dann weiterentwickelt. Meine Überlegung war damals aber schn, dass man Tests die mehrere "Units" beeinflussen separat definiert. In der Logik, ja, würde so ein Test mit mehreren Decoratorn schon in `DecoratorTest` gehören

Reply:
https://github.com/Grishnackh/akhq/pull/13/commits/816e32148e58b7640bf4c67b2ce641e2ccfb8376 erweitert die Testklasse, um einen Test der das Decorator dekorieren testet. Der Testcode ist jetzt doch in der `AvroWireFormattedRecordTest`, wegen der Abhängigkeiten zu `serializeAvro()` und `MyRecord`. @Grishnackh lass den Test bei dir bitte mal noch durchlaufen. Mein Unit-Test setup funktioniert bei mir aus irgendeinem Grund nicht mehr...

Reply:
Streng genommen wäre der DecoratorsTest damit wahrscheinlich kein Unit-Test mehr sondern schon ein Integrationstest.